spec/collectionspace/converter/publicart/acquisition_spec.rb

```diff
@@ -40,16 +40,21 @@
     end
 
     context 'fields overridden by publicart' do
-      context 'when local place' do
```

Comment:
@Paulmulonzia - Where possible (when field is repeatable), the maximally populated sample data (first row of CSV) should contain at least one value from each possible authority. Then we do not need to write separate tests for "when local place," etc.

If a field is not repeatable but may be populated by multiple authorities, we will need to write a separate test for each authority, with a separate row of the spreadsheet for each, to make sure each column is getting converted properly.

I updated the sample data CSV and fixture XML to reflect this, then fixed the logical structure of the test file.